packages/vite-plugin-shopify-defer-css/README.md

```diff
@@ -0,0 +1,31 @@
+# vite-plugin-shopify-defer-css
+
+Defer loading non-critical stylesheets to improve the performance of your online store. [Learn more](https://web.dev/articles/defer-non-critical-css).
+
+## Usage
+
+```js
+// vite.config.js
+import deferCss from 'vite-plugin-shopify-defer-css'
+
+export default {
+  plugins: [
+    /* Plugin options are not required, defaults shown */
+    deferCss({
```

Comment:
@tuanpham-barrel extracted this feature into its plugin

Reply:
This will require double work for config if user has different path. Because for many people, putting js/css files in `src` makes more sense than putting them in `frontend`.

Reply:
No configuration is required here; the defaults match the default directories of vite-plugin-shopify. If needed, you would configure sourceCodeDir and entrypointsDir.

Reply:
So if I want to put js/css file in `src` folder, do I have add the same settings `sourceCodeDir` and `entrypointsDir` into 2 places?

Reply:
So if I want to put js/css file in `src` folder, do I have add the same settings `sourceCodeDir` and `entrypointsDir` into 2 places?

Reply:
Yes please 🙏 it would be just 2 settings to adjust